iceoryx_posh/source/roudi/port_manager.cpp

```diff
@@ -53,13 +53,13 @@ PortManager::PortManager(RouDiMemoryInterface* roudiMemoryInterface) noexcept
     }
```

Comment:
@elBoberido Was the `chmod` on this file on purpose?

Reply:
Actually no. I have no idea how I did this but it actually makes sense. All the other files also have `644` and the executable does not make sense for a source file